posts/2017-01-05-dat-next.md

```diff
@@ -14,23 +16,43 @@ npm install -g dat-next
 
 Once you get it installed run `dat` to see the usage guide or view more docs [on Github](https://github.com/joehand/dat-next).
 
+## New Commands:
+
+### Clone
+
+`dat clone` provides a git-like command to download remote Dat archives:
+
+![screen shot 2017-01-06 at 10 11 45](https://cloud.githubusercontent.com/assets/684965/21727956/9fdc561e-d3f8-11e6-8710-f2cae09fbaff.png)
+
+### Init or Create
+
+`dat init` and `dat create` creates a new Dat archive and imports all the files for you, getting it ready to share:
```

Comment:
I'm ok with aliasing in general, but perhaps it adds a bit too much unnecessary confusion to introduce both concepts here? maybe we can just pick one because in later references we're then mentioning both init/create and it can become unclear if they're different commands or not